Automattic/jetpack
# AAG: add Activity Log data to Activity card

This is a replay of #12468.

#### Changes proposed in this Pull Request:
This commit:

- adds the `v4/site/activity` endpoint to the API package.
- adds a new `QuerySiteActivity` to fetch activity data.
- displays the data in the Activity card.

#### Jetpack product discussion

* Internal reference: p1630603072207500-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start with a site using a Backup plan or above, connected to WordPress.com and where you've added credentials for your site.
* Go to Jetpack > Dashboard
* Check the Activity card towards the bottom of the page.
